src/session/content/chat_history.rs

```diff
@@ -109,6 +110,27 @@ mod imp {
     impl BinImpl for ChatHistory {}
 }
 
+fn subtitle_expression(chat: &Chat) -> gtk::Expression {
+    match chat.type_() {
+        ChatType::Private(data) => {
+            let user_id = data.id();
+            let my_id = chat.session().me().unwrap().id();
+            if user_id != my_id {
+                let user_expression = gtk::ConstantExpression::new(data);
+                User::formatted_status_expression(&user_expression)
+            } else {
+                gtk::ConstantExpression::new(&"").upcast()
+            }
+        }
+        ChatType::Supergroup(data) => Supergroup::member_count_expression(data),
+        ChatType::BasicGroup(data) => BasicGroup::member_count_expression(data),
```

Comment:
These methods belong to a type. According to convention, they are better used with "." operator
```suggestion
        ChatType::Supergroup(data) => data.member_count_expression(),
        ChatType::BasicGroup(data) => data.member_count_expression(),
```